tests/UserAgentsTest/V3/FullTest.php

```diff
@@ -73,45 +73,54 @@ public static function setUpBeforeClass() : void
 
         $version = (string) $buildNumber;
 
-        $logger               = new NullLogger();
```

Comment:
Instead of these changes, could we add a new build stage, where we just call the CLI build command?

Reply:
These blocks of code vary slightly for V3/V4, how would that be handled in the CLI Command?

Reply:
The code you moved into the `try` block is nearly the same, as in the CLI build command.

Instead of changing 6 files, I would prefer to add a new step in the build pipeline (travis), where we just run the CLI build command.

Reply:
Well, this isn't just for Travis runs, I'd make use of it when running tests locally as well.

Reply:
If you think it makes sense, I could try to call the build command and remove a lot of the duplicated code between the three (or 6, though this seems trickier to do with a single build command unless it branches a bit based on the installed browscap version) test case files.

I know it's possible to call symfony commands from other commands, but not sure how simple it is from non-console code.